scripts/package-emulator.sh

```diff
@@ -1,33 +1,45 @@
 #!/bin/bash
-set -
-cd ..
-. setup.sh
+source build/envsetup.sh
+HOST_ARCH=$(get_build_var HOST_ARCH)
+lunch "aosp_${HOST_ARCH}-userdebug"
```

Comment:
nit: In case the build machine arch didn't match the built emulator, this won't work.